esp/src/src-react/components/WorkunitSummary.tsx

```diff
@@ -69,7 +72,35 @@ export const WorkunitSummary: React.FunctionComponent<WorkunitSummaryProps> = ({
         }, [workunit])
     });
 
+    const nextWuid = React.useCallback((wuids: WUQuery.ECLWorkunit[]) => {
+        for (const wu of wuids) {
+            if (wu.Wuid !== wuid) {
+                pushUrl(`/workunits/${wu.Wuid}`);
+                break;
+            }
+        }
+
+    }, [wuid]);
+
     const buttons = React.useMemo((): ICommandBarItemProps[] => [
+        {
+            key: "previous", iconOnly: true, tooltipHostProps: { content: nlsHPCC.previousWorkunit }, iconProps: { iconName: "Previous" },
```

Comment:
I missed it when I was looking earlier, but lint was complaining about the casing of nlsHPCC.previousWorkunit & nextWorkunit